group_vars/recap/production.yml

```diff
@@ -54,4 +37,4 @@ datadog_checks:
         source: recap
         sourcecategory: recap
     tags: "recap, environment:production, role:recap"
-drupal_major_version: 8
\ No newline at end of file
+drupal_major_version: 8
```

Comment:
@kayiwa This should be drupal 9 here. I realize that it is wrong already, but would you mind updating it. I just do not want running the playbook to break something.